specification/columns/skudetails.md

```diff
@@ -0,0 +1,50 @@
+# SKU Details
+
+The **SKU Details** column represents a list of relevant properties shared by all resources having the same [**SKU ID**](#skuid) which are not captured in another FOCUS column. These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics.
+
+
+The _SkuDetails_ column adheres to the following requirements:
+
+* The properties (both key and value) contained in the _SkuDetails_ column **MUST** be shared across all resources having the same _SkuId_.
```

Comment:
```suggestion
* The properties (both key and value) contained in the _SkuDetails_ column MUST be shared across all resources having the same _SkuId_.
```

Reply:
@kk09v, although the group has not confirmed the final editorial rules, there was an agreement about don't use bold for the normative keywords.